pom.xml

```diff
@@ -22,15 +22,15 @@
 
   <groupId>io.cdap.delta</groupId>
   <artifactId>bigquery-delta-plugins</artifactId>
-  <version>0.1.0-SNAPSHOT</version>
+  <version>0.1.0</version>
   <name>BigQuery Delta plugins</name>
   <packaging>jar</packaging>
 
   <properties>
     <avro.version>1.8.2</avro.version>
     <bigquery.version>1.78.0</bigquery.version>
-    <cdap.version>6.0.0</cdap.version>
-    <delta.version>0.1.0-SNAPSHOT</delta.version>
+    <cdap.version>6.2.0</cdap.version>
```

Comment:
when bumping versions, don't change the cdap.version dependency unless it's a snapshot version.